src/java-interop/java-interop-gc-bridge-mono.cc

```diff
@@ -380,10 +384,10 @@ java_interop_gc_bridge_register_bridgeable_type (
 	MonoJavaGCBridgeInfo   *info    = &bridge->mono_java_gc_bridge_info [i];
 
 	info->klass             = mono_class_from_mono_type (type);
-	info->handle            = mono_class_get_field_from_name (info->klass,     "handle");
-	info->handle_type       = mono_class_get_field_from_name (info->klass,     "handle_type");
-	info->refs_added        = mono_class_get_field_from_name (info->klass,     "refs_added");
-	info->weak_handle       = mono_class_get_field_from_name (info->klass,     "weak_handle");
+	info->handle            = mono_class_get_field_from_name (info->klass,     (char*)"handle");
```

Comment:
for this and similar casts before, please use `static_cast` or `reinterpret_cast`, as necessary

Reply:
In general, in modern C++ the string constants should be defined as

```c++
const char my_constant[] = "stuff";
```

Reply:
ok, let's use the `const_cast<>` then